kubespray_deploy/kubespray_deploy_script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
align naming with api, if we have cluster api, lets have scripts/cluster

Reply:
Fixed 249493c